server/sonar-process-monitor/src/main/java/org/sonar/application/config/JdbcSettings.java

```diff
@@ -34,13 +34,20 @@
 
 import static org.apache.commons.lang.StringUtils.isEmpty;
 import static org.apache.commons.lang.StringUtils.isNotEmpty;
-import static org.sonar.api.database.DatabaseProperties.PROP_EMBEDDED_PORT;
-import static org.sonar.api.database.DatabaseProperties.PROP_EMBEDDED_PORT_DEFAULT_VALUE;
-import static org.sonar.api.database.DatabaseProperties.PROP_URL;
 import static org.sonar.process.ProcessProperties.JDBC_URL;
 
 public class JdbcSettings {
 
+  private static final String PROP_URL = "sonar.jdbc.url";
+  private static final String PROP_DRIVER = "sonar.jdbc.driverClassName";
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused "PROP_DRIVER" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1068)